hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMSecurityProtocolServer.java

```diff
@@ -93,15 +100,22 @@ public class SCMSecurityProtocolServer implements SCMSecurityProtocol {
   private final ProtocolMessageMetrics metrics;
   private final StorageContainerManager storageContainerManager;
 
+  // SecretKey may not be enabled if when neither block token nor container
+  // token is enabled.
+  private final SecretKeyManager secretKeyManager;
+
   SCMSecurityProtocolServer(OzoneConfiguration conf,
-      CertificateServer rootCertificateServer,
-      CertificateServer scmCertificateServer,
-      X509Certificate rootCACert, StorageContainerManager scm)
+                            CertificateServer rootCertificateServer,
```

Comment:
Minor note and hint here: this is an unnecessary indentation change. You can disable it by editor/codestyle/java -> wrapping and braces tab and disabling "align when multiline" on method call arguments.

Reply:
Thanks, that's helpful. :)